src/services/chat.service.js

```diff
@@ -0,0 +1,29 @@
+import { Op } from 'sequelize';
+import db from '../database/models/index';
+
+const ChatService = {
+  async addChat(chat) {
+    const newChat = await db.Chats.create(chat);
+    return newChat;
+  },
+
+  async getChat(userId, offset, limit) {
+    const chats = await db.Chats.findAll({
+      where: {
+        [Op.or]: [
+          { authorId: userId },
+          { receiverId: null },
+          { receiverId: userId },
```

Comment:
Why do we have these?

Reply:
because we need to get all the chats:
- sent by the user 
- received by the user 
- which are public